tests/mortar/tests.h

```diff
@@ -470,3 +470,323 @@ class StokesOperator
 private:
   const double delta_1_scaling;
 };
+
+
+/**
+ * @brief Base class for the operator base
+ */
+template <int dim,
+          typename Number,
+          typename VectorizedArrayType = VectorizedArray<Number>>
+class GeneralStokesOperator : public Subscriptor
+{
+public:
+  using FECellIntegrator =
+    FEEvaluation<dim, -1, 0, dim + 1, Number, VectorizedArrayType>;
+
+  using VectorType = LinearAlgebra::distributed::Vector<Number>;
+
+
+  GeneralStokesOperator(const MappingQ<dim>             &mapping,
```

Comment:
I want to test mixed elements here: `FE_Q(k+1)^dim+FE_Q(k)` and `FE_Q(k+1)^dim+FE_DGP(k)`. But this will happen in a follow-up PR.